http_check/datadog_checks/http_check/http_check.py

```diff
@@ -243,8 +243,10 @@ def send_status_down(loginfo, down_msg):
             tags_list = list(tags)
             tags_list.append('url:{}'.format(addr))
             tags_list.append("instance:{}".format(instance_name))
-            self.gauge('http.ssl.days_left', days_left, tags=tags_list)
-            self.gauge('http.ssl.seconds_left', seconds_left, tags=tags_list)
+            if days_left >= 0:
```

Comment:
But you'll want to report this as a negative number when that is the right value. You could change the returned value to `None` instead and check for that:

`return AgentCheck.CRITICAL, None, None, msg`

`if days_left is not None:`